solpolpy/polarizers.py

```diff
@@ -19,6 +19,11 @@ def npol_to_mzp(input_cube):
     in_list = list(input_cube)
     conv_fact = (np.pi * u.radian) / (180 * u.degree)
 
+    if input_cube['angle_1'].meta['OBSRVTRY'] == 'STEREO_B':
+        offset_angle = -18 * u.degree * conv_fact  # STEREOB
```

Comment:
These factors (-18, 45.8) are a bit cryptic to me. We should document them somehow more clearly. Where do they come from? What do they do?

Reply:
These are  obtained from [Thompson (2015)](https://www.sciencedirect.com/science/article/pii/S0019103515003620?via%3Dihub)

Reply:
Please note this in the source code and the docstring fro this function.